arrow-libs/optics/arrow-optics-compose/src/commonMain/kotlin/arrow/optics/Copy.kt

```diff
@@ -0,0 +1,29 @@
+package arrow.optics
+
+import androidx.compose.runtime.MutableState
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+
+/**
+ * Modifies the value in this [MutableState]
+ * by applying the function [block] to the current value.
+ */
+public inline fun <T> MutableState<T>.update(block: (T) -> T) {
+  value = block(value)
+}
```

Comment:
Oh, I thought it had `update` out of the box but it seems `androidx.compose.runtime.MutableState` is not an Atomic reference like `MutableStateFlow`.

IMO, we can keep or leave this `update` in favor of `setValue`.